packages/s3-static-assets/tests/upload-assets.test.ts

```diff
@@ -180,3 +192,35 @@ describe.each`
     });
   }
 );
+
+describe("Content upload options", () => {
+  describe.each([
```

Comment:
Could you use [this way ](https://github.com/danielcondemarin/serverless-next.js/blob/master/packages/s3-static-assets/tests/upload-assets.test.ts#L95) of testing with `describe.each` to keep it consistent with other packages?

Reply:
Meaning remove the nesting and simply keep at the root?

Reply:
I mean not to use the square bracket notation and instead use string interpolation 
```
describe.each`
...
`
```

Reply:
Right, I don't know why I thought it was about the nesting 😄